pkg/controllers/management/kontainerdrivermetadata/metadata_handler.go

```diff
@@ -10,27 +10,25 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/rancher/norman/types/convert"
+	v3Setting "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
```

Comment:
```suggestion
	v3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
```
I would keep this as `v3`